samples/nrf9160/nrf_cloud_mqtt_multi_service/child_image/mcuboot/boards/nrf9160dk_nrf9160.conf

```diff
@@ -12,3 +12,7 @@ CONFIG_SPI_NRFX_RAM_BUFFER_SIZE=32
 CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK=y
 
 CONFIG_FPROTECT=y
+
+# Enabling SPI increases the MCUBoot image size such that it won't fit in the default secondary
+# partition size (0xC000). The minimum required size is 0xD000
```

Comment:
```suggestion
# Enabling SPI increases the MCUBoot image size so that it does not fit in the default secondary
# partition (0xC000). The minimum required size is 0xD000
```